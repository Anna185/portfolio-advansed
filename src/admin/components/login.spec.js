import login from './Login.vue';
import {shallowMount} from "@vue/test-utils";


describe('На странице есть кнопка  Авторизоваться', () => {
  it ('Ищем кнопку авторизации по классу', () => {

    const wrapper = shallowMount(login);
    const button = wrapper.find(".login__send-data");

    expect(button.element).toMatchSnapshot();


  });

  it ('Ищем кнопку авторизации по слову  Авторизоваться', () => {

    const wrapper = shallowMount(login);
    const button = wrapper.findAll("button").at(2);

    expect(button.text()).toBe('Авторизоваться');

  });
});

