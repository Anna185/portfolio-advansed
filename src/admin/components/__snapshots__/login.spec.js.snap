// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`На странице есть кнопка  Авторизоваться Ищем кнопку авторизации по классу 1`] = `
<button
  class="login__send-data"
  type="submit"
>
  Отправить
</button>
`;
